NitroxClient/Communication/Packets/Processors/VehicleUndockingProcessor.cs

```diff
@@ -31,26 +35,35 @@ public override void Process(VehicleUndocking packet)
             
             using (packetSender.Suppress<VehicleUndocking>())
             {
-                vehicles.SetOnPilotMode(packet.VehicleId, packet.PlayerId, true);
-                vehicleDockingBay.subRoot.BroadcastMessage("OnLaunchBayOpening", SendMessageOptions.DontRequireReceiver);
-                SkyEnvironmentChanged.Broadcast(vehicleGo, (GameObject)null);
-
-                
-
-                vehicle.StartCoroutine(WaitBeforePushDown(vehicle, vehicleDockingBay));
-                
+                Optional<RemotePlayer> player = remotePlayerManager.Find(packet.PlayerId);
+                if (packet.UndockingStart)
```

Comment:
minor: can we break this into two functions (undocking/docking) related logic?

Reply:
Maybe i should reword the boolean "UndockingStart". Both parts are undocking related logic. The first part starts the undocking process (when player enters vehicle) and second part (UndockingStart=false) when the undocking is finished (the real undocking)
I can separate both anyway.